tests/cupy_tests/test_cublas.py

```diff
@@ -453,12 +453,23 @@ def test_dgmm_inplace(self, dtype):
         cublas.dgmm(self.side, self.a, self.x, out=self.a)
         cupy.testing.assert_allclose(self.a, ref, rtol=self.tol, atol=self.tol)
 
-    @pytest.mark.skipif(cupy.cuda.runtime.is_hip,
-                        reason="HIP fails for some condition")
+    _dgmm_incx_minus_one_hip_skip_condition = [
+        ('C', 'F', (9, 10), 'R'),
+        ('C', 'F', (10, 9), 'R'),
+        ('F', 'F', (9, 10), 'L'),
+        ('F', 'F', (10, 9), 'L'),
```

Comment:
Seems like it fails only when c is 'F' order is involved regardless of the other parameters?

Reply:
When I dug into how the API was called, the condition was (#4540)
- mode=0
- incx=-1

Not sure how this maps to that c is in  the `F` order.

Reply:
There are tests that pass even when c is 'F'. I can't find an apparent rule there.